tests/ci/setup.sh

```diff
@@ -115,6 +115,7 @@ build_dokku_docker_image() {
 
 run_dokku_container() {
   echo "=====> run_dokku_container on CIRCLE_NODE_INDEX: $CIRCLE_NODE_INDEX"
+  echo "redirect: https://github.com/dokku/dokku-redirect.git" >>/var/lib/dokku/plugin-list
```

Comment:
I think maybe we need to create that directory prior to setup? Maybe try creating via `sudo`?

Reply:
Do you know if sudo can be used in that context?

I'm thinking of doing a `| sudo tee /var/lib/dokku/plugin-list` instead.

Reply:
Yeah, try that.